### PR TITLE
Fix png sizing

### PR DESCRIPTION
**Pull request by jhyching**
**What this PR does / why we need it**:
Fix bad cropping of `png` and add a bit more functionality.

**Which issue(s) this PR fixes**
Fixes bad cropping in `png` introduced by changes to chrome webdriver.

**Special notes for your reviewer**:

**Release note**:
```release-note
Switch custom cropping function to the one provided by Bokeh, as well as enabling some of  the extra functionality.
```

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,13 +1,14 @@
 [tox]
-envlist = py38, py39, py310
+#envlist = py38, py39, py310
```
Comment (landryzs) [inline tox.ini:2]: Any reasoning behind commenting out vs. deleting?

**Comment by tjofas** [comment]
Removed Python3.8 and added 3.11 in some different places, prepped for a release 5.0.0 (as we are dropping support for 3.8) and added support to use make black for code formatting.

Is it possible for you @jhyching to change the target of this PR to the branch `Release-5.0.0`? Otherwise let's open a new PR targeting this branch. That way we adhere to the workflow from Pelle to first merge into a release branch and then do the release when merging that branch to master

**Review by jhyching — COMMENTED** [review]
LGTM 👍

File: tox.ini (hunk 1 above)
Comment (jhyching) [reply tox.ini]: No reason just didn't notice it 😅

**Review by tjofas — APPROVED** [review]

**Pull request merged**